static/json/datadog-agent-ecs-eu.json

```diff
@@ -0,0 +1,101 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "datadog-agent",
+      "image": "datadog/agent:latest",
+      "cpu": 10,
+      "memory": 256,
```

Comment:
too bad json does not support comments, I'd love to add a "customize according to observed usage". Is that something we could add to the doc that links to it?

Reply:
we could indeed, I'm going to add it.